portal/src/main/webapp/js/src/DataProxyFactory.js

```diff
@@ -41,7 +41,7 @@ var DataProxyFactory = (function()
 	 *
 	 * @return singleton instance of MutationDataProxy class
 	 */
-	function getDefaultMutationDataProxy()
+	function getDefaultMutationDataProxy(geneticProfiles,caseSetId,caseIdsKey,caseList)
```

Comment:
There wouldn't be any use case of the function with no parameter. Since from now all the tabs(plots,coexp, mutation etc) should be able to support single and multiple studies. To reduce code complexity(implementing different ways for single and multiple studies), its better to go with one parameterised function